uk.co.compucorp.civicrm.hremails/email-templates/src/pages/leave-request.html

```diff
@@ -0,0 +1,69 @@
+---
+layout: leave-absences
```

Comment:
Not a comment, you use that in the top of the file to specify not only the layout that the email should use, but also to pass the value of placeholders contained in the layout